src/model/schema/ThunderstoreSchema.ts

```diff
@@ -0,0 +1,131 @@
+import Ajv from "ajv";
+import addFormats from "ajv-formats";
+import schema from "../../../schema.json";
+import schemaSchema from "../../../schemaSchema.json";
```

Comment:
The naming confuses me. Would renaming schema to e.g. "ecosystemData" and schemaSchema to "ecosystemSchema" make more sense?

Reply:
Hah yeah, these shouldn't have been committed with these names. schemaSchema is dumb but it's the JSON schema to the schema, with the schema itself being just schema. Maybe `ecosystemSchema.json` and `ecosystemJsonSchema.json`? It's awkward.

Reply:
> schemaSchema is dumb but it's the JSON schema to the schema, with the schema itself being just schema

I did some quick googling that somewhat confirmed my understanding that in this context "schema" means something that describes the structure of the data. So I'd argue the `schema.json` doesn't contain a schema, but the data itself. Whereas `schemaSchema.json` describes the structure of the data, i.e. it is a schema.

What do you mean when you say schema?

Reply:
"schema" unfortunately refers to both the stuff in the ecosystem schema *and* the data format of the ecosystem schema. What I've done here (and I'm open for better names) is renamed the two files as such:
- schema.json -> ecosystem.json
- schemaSchema.json -> ecosystemJsonSchema.json

Reply:
If I understood your argument correctly, the confusion we now have in this repo is the fact we use "ecosystem schema" to describe the whole contents of the other repo, which contains both the data and the schema that describes the data?

Anyway, I like the renaming of the files, but could we also rename the local variables to make the code in this file less confusing?